docs/nodes/validate/add-a-validator.md

```diff
@@ -48,182 +48,308 @@ The response has your node’s ID:
+`startTime` and `endTime` are required to specify the time of starting/leaving validation. The minimum duration that one can validate the Primary Network is 2 weeks, and the maximum duration is one year. One can re-enter the Primary Network after leaving, it’s just that the maximum _continuous_ duration is one year. `startTime` and `endTime` are the Unix times when your validator will start and stop validating the Primary Network, respectively. `startTime` must be in the future relative to the time the transaction is issued.
+
+The sample code uses `const startTime: BN = UnixNow().add(new BN(60 * 1))` and
+`const endTime: BN = startTime.add(new BN(26300000))` to compute the Unix time 1 minute and 304 days in the future to use as the values of `startTime` and `endTime`, respectively.
+
+:::tip
+You can create your own unix timestamp [here](https://www.unixtimestamp.com/) or by using the `UnixNow()` method
+:::
+
+To create your own start times, please follow the steps below:
+
+Locate this line in the file
+
+```ts
+const startTime: BN = UnixNow().add(new BN(60 * 1));
+const endTime: BN = startTime.add(new BN(26300000));
+```
+
+Change `startTime` and `endTime` to new `BN` values, for example:
+
+```ts
+const startTime: BN = new BN(1654656829); // Wed Jun 08 2022 02:53:49 GMT+0000
+const endTime: BN = new BN(1662602029); // Thu Sep 08 2022 01:53:49 GMT+0000
+```
+
+#### Delegation Fee Rate
+
+Avalanche allows for delegation of stake. This parameter is the percent fee this validator charges when others delegate stake to them. For example, if `delegationFeeRate` is `10` and someone delegates to this validator, then when the delegation period is over, 10% of the reward goes to the validator and the rest goes to the delegator.
+
+#### Stake Amount
+
+In order to validate the Fuji Network, one must stake AVAX.
+
+The sample code assigns the variable, `stakeAmount` to be the minimum amount of AVAX that a node must stake to become a validator(1 AVAX).
+
+```ts
+const stakeAmount: any = await pchain.getMinStake();
+```
+
+To stake a different amount, please replace the code above with a stake amount of 1 AVAX or greater. Below sets a new value of 10,000 AVAX (`10000000000000` GWEI). Value is set in GWEI format where `1,000,000,000` GWEI = 1 AVAX.
+
+Please follow the Example below for steps to change the stake amount:
+
+Locate this line in the file
+
+```js
+  const unsignedTx: UnsignedTx = await pchain.buildAddValidatorTx(
+    ...
+    stakeAmount.minValidatorStake,
+    ...
+  )
+```
+
+and change the default value to a new `BN` value, for example:
+
+```js
+  const unsignedTx: UnsignedTx = await pchain.buildAddValidatorTx(
+    ...
+    new BN(1e12),
+    ...
+  )
+```
```

Comment:
This is a convoluted way to say that they just need to put a new value for the stake amount as BN(1e12). Please rewrite this section to make it short to the point that they just need to provide a correct amount to the API.

Reply:
I have read this multiple times and I personally feel that this is short and clear. I have removed some language but am open to more suggestions.

Reply:
This whole section can be reduced to one paragraph/sentence like this:

Set the the proper staking amount when calling pchain.buildAddValidatorTx by replacing `stakeAmount.minValidatorStake` with a number such as `BN(1e12)` which is 10,000 AVAX, in the example code.